wgsl/index.bs

```diff
@@ -942,12 +942,14 @@ An <dfn noexport>array</dfn> is an indexable grouping of element values.
     <tr><th>Type<th>Description
   </thead>
   <tr><td algorithm="sized array type">array<|E|,|N|><td>An |N|-element array of elements of type |E|.<br>
-                    |N| must be 1 or larger.
   <tr><td algorithm="runtime-sized array type">array<|E|><td>A <dfn noexport>runtime-sized</dfn> array of elements of type |E|,
                        also known as a runtime array.
                        These may only appear in specific contexts.<br>
 </table>
 
+When specified, an array size |N|
+must be an integer literal or the name of [=pipeline-overridable=] constant,
```

Comment:
Why require it to be overridable? As with the [the `workgroup_size` attribute](https://github.com/gpuweb/gpuweb/pull/1742), I think we should allow any module-scope constant.

Reply:
That makes sense to me.
Pipeline-overridable is the hard case.
And it's still it doesn't stray into const_expr (#1272) yet, but leaves that door open.

Reply:
Updated. PTAL.